app/controllers/webo_track/webousers_controller.rb

```diff
@@ -0,0 +1,38 @@
+module WeboTrack
+  class WebousersController < ApplicationController
+    def new
+      @webouser = Webouser.new
+    end
+
+    def create
+      @webouser = Webouser.new(user_params)
+
+      if @webouser.save
+        redirect_to users_path, notice: "Blog type was successfully created."
+      else
+        render :new, status: :unprocessable_entity
+      end
+    end
+
+    def index
+      @webousers = Webouser.all
+    end
+
+    def assign_asset_edit
+      @webouser = Webouser.find(params[:id])
+      @assets = Asset.all
+    end
+
+    def assign_asset_update
```

Comment:
+1 - Not meaningful.
assign_asset_update but updating webouser record. 
Webouser should be WeboUser. It's fine for now.